From edbf81500178eb815a87d9812d59b09032732e5f Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Thu, 2 May 2024 18:04:29 +0300
Subject: [PATCH] Remove fancy-pypi-readme

---
 pyproject.toml | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index ffc98fd..87f54fe 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -94,7 +94,7 @@ typecheck = { chain = [
 "typecheck:mypy" = "mypy ."
 
 [build-system]
-requires = ["hatchling", "hatch-fancy-pypi-readme"]
+requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [tool.hatch.build]
@@ -105,17 +105,6 @@ include = [
 [tool.hatch.build.targets.wheel]
 packages = ["src/openai"]
 
-[tool.hatch.metadata.hooks.fancy-pypi-readme]
-content-type = "text/markdown"
-
-[[tool.hatch.metadata.hooks.fancy-pypi-readme.fragments]]
-path = "README.md"
-
-[[tool.hatch.metadata.hooks.fancy-pypi-readme.substitutions]]
-# replace relative links with absolute links
-pattern = '\[(.+?)\]\(((?!https?://)\S+?)\)'
-replacement = '[\1](https://github.com/openai/openai-python/tree/main/\g<2>)'
-
 [tool.black]
 line-length = 120
 target-version = ["py37"]
-- 
2.44.0

